### PR TITLE
stable 2020-11-20 workbooktemplates

### PR DESCRIPTION
**Pull request by vutran01**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

Update workbookTemplates api-version to be stable version

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet. Already published to public regions
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet. New api-version already published
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [x] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by kairu-ms — CHANGES_REQUESTED** [review]

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+    },
+    "WorkbookError": {
+      "description": "Error message that will indicate why the operation failed.",
+      "properties": {
+        "error": {
+          "description": "Error message object that will indicate why the operation failed.",
+          "$ref": "#/definitions/WorkbookErrorBody"
+        }
+      }
+    },
```
[the remark this answers is not in the document]
Comment (vutran01) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json:441]: rename WorkbookError to WorkbookTemplateError

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/examples/WorkbookTemplateAdd.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        "galleries": [
+          {
+            "name": "Simple Template",
+            "category": "Failures",
+            "type": "tsg",
+            "resourceType": "microsoft.insights/components",
+            "order": 100
+          }
+        ],
+        "templateData": {}
```
Comment (kairu-ms) [inline specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/examples/WorkbookTemplateAdd.json:21]: Can you add an example for "templateData" property?

**Review by kairu-ms — CHANGES_REQUESTED** [review]

**Comment by kairu-ms** [comment]
@vutran01 This PR needs ARM review, please follow the `ARM API Review Checklist` to request ARM review. Thanks.

**Comment by chiragg4u** [comment]
Please go through the contribution checklist for ARM API Review added as part of the review and mark them completed. It will help you fix most of the issues and improve the quality of swagger and help close the PR faster. This is a requirement before review can be continued.

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/examples/WorkbookTemplateAdd.json (hunk 2 above)
Comment (vutran01) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/examples/WorkbookTemplateAdd.json:21]: "templateData" is the workbook template itself. I'll add an example

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+          "description": "Service-defined error code. This code serves as a sub-status for the HTTP error code specified in the response."
+        },
+        "message": {
+          "type": "string",
+          "description": "Human-readable representation of the error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorFieldContract"
```
[the remark this answers is not in the document]
Comment (vutran01) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json]: changed name to WorkbookTemplateErrorFieldContract

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a workbook for an Azure resource.",
+          "$ref": "#/definitions/WorkbookTemplateProperties"
+        }
+      },
+      "description": "The parameters that can be provided when updating workbook template."
+    },
+    "ErrorFieldContract": {
```
[the remark this answers is not in the document]
Comment (vutran01) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json]: changed name to WorkbookTemplateErrorFieldContract

**Comment by kairu-ms** [comment]
Please fix conflicts. Thanks.

**Review by kairu-ms — COMMENTED** [review]
Please Fix CI issues. Thanks

**Review by vutran01 — COMMENTED** [review]
CI has been addressed

**Review by kairu-ms — APPROVED** [review]

**Comment by kairu-ms** [comment]
Your PR needs ARM review.

**Pull request merged**